app/src/main/java/io/apicurio/registry/storage/impl/AbstractMapRegistryStorage.java

```diff
@@ -199,8 +200,6 @@ protected ArtifactMetaDataDto createOrUpdateArtifact(String artifactId, Artifact
         contents.put(MetaDataKeys.CREATED_ON, currentTimeMillis);
         contents.put(MetaDataKeys.MODIFIED_ON, currentTimeMillis);
```

Comment:
I don't think this is useful meta-data to put on a version.  The **artifact** has a logical `modifiedOn` ... but similar to the `createdOn` value for an artifact, the `modifiedOn` for an artifact should be the `createdOn` value of the artifact's most recent version.

Reply:
I can fix it, but nothing has changed regarding the `modifiedOn` metadata here.